docs/alerts/index.md

```diff
@@ -38,3 +40,71 @@ Anomaly | yes | yes | yes
+
+![Create Alert Rules](../images/alerts/create-alert-rules.gif)
+
+### Alert Metric Tab
+
+If you open the Create Alert Rule page from a specific App, that particular App will be automatically selected by default in the dialog. Next to the App selection, you'll find a dropdown where you can choose one or multiple metrics you wish to receive alerts for.
+
+#### Transformation
+
+Transformations are used to modify metrics using mathematical expressions and functions. For example, imagine you want to get alerted when both `request.size` and `response.size` together exceed some threshold or become anomalous.  You would then transform them into a single dataseries by using an expression like this:
+
+`request.size + response.size`
+
+And then you would create an alert on this new data series.
+
+Read about [Transformations](../dashboards/chart-builder/#transformation) to learn more about transformations, functions, and expressions used to perform transformations.
+
+####  Filter by
+
+The Filter section is used to define criteria that will trigger an alert only when specific conditions are met.
+Suppose you collect data from multiple hosts but you'd like to create an alert for a specific host only.
+You would choose tag name `os.host` and single value `host-x`.
+
+####  Group by
+
+In the Group by section you choose what data series to group on. For example, if you
+choose the `os.host` tag and for the aggregation in `Aggregation` field you choose `all separately`
+the alert will trigger for each host separately.
+
+####  Rollup by
+
+Data points sent to Sematext Cloud have a time label. If a metric is updated every 10 seconds, and you are
+looking at a 24-hour time range, you need 14,400 points to display all data points.. Rollup by is a function used to group data points in time buckets. So the metrics coming in within the specified time window of the alert (which is set in the condition tab) will be aggregated by the selected function in this section.
+
+Refer to [Rollup By](../dashboards/chart-builder/#rollup-by) to learn more.
+
+### Alert Condition Tab
+
+This is where you select the [alert type](https://sematext.com/docs/alerts/#alert-types) and the priority. If you select threshold type alert you need to enter the threshold value to trigger and alert and specify a time window.
+
+#### Ignore regularly occuring spikes and dips 
+
+Where we look at historical data and try to decide if an alert fits the regular stream of spikes and dips in metrics. If it does, we won't interrupt you with notifications.
+Based on the filter combination we find all previous spikes and dips from the past. Split them into group-by values (+-5% we assume as the same), find LCM (least common multiple) in the groups. This means we're trying to include the current value into our LCM groups to check if it fits any of them or not. If it fits, we assume this is a regular event and ignore it.
```

Comment:
Split them... ==> We split them?
we assume as the same ==> we treat as the same